rotkehlchen/chain/evm/decoding/velodrome/velodrome_cache.py

```diff
@@ -164,15 +169,28 @@ def query_velodrome_data_from_chain_and_maybe_create_tokens(
     if inquirer.chain_id == ChainID.OPTIMISM:
         data_contract = inquirer.contracts.contract(VELODROME_SUGAR_V2_CONTRACT)
         protocol = VELODROME_POOL_PROTOCOL
+        counterparty = CPT_VELODROME
     else:
         data_contract = inquirer.contracts.contract(AERODROME_SUGAR_V2_CONTRACT)
         protocol = AERODROME_POOL_PROTOCOL
+        counterparty = CPT_AERODROME
 
-    pool_data = []
+    pool_data: list[dict] = []
     offset = 0
     limit = 200
     pool_data_chunk: list[dict] = []
+    last_notified_ts = Timestamp(0)
+    chain_name = inquirer.chain_id.to_name()
```

Comment:
same here for compression of initialization. Too many stuff are initialized in all those lines before the while